subscription_manager.go

```diff
@@ -971,7 +972,8 @@ func parseCipherInterface(data interface{}, pnConf *Config, module crypto.Crypto
 					pnConf.Log.Println("v[pn_other]", v["pn_other"], v, msg)
 					decrypted, errDecryption := decryptString(module, msg)
 					if errDecryption != nil {
-						pnConf.Log.Println(errDecryption, msg)
+					pnConf.Log.Println(errDecryption, msg)
```

Comment:
Is this on purpose to have two separate Println statements?

Reply:
I wanted to add this information as an additional thing. 
I can remove it if you want.

Reply:
merged